test/__snapshots__/modules-option.test.js.snap

```diff
@@ -498,49 +498,49 @@ exports[`"modules" option issue #995 #2: result 1`] = `
 /* Local */
 
 /* class="😀" */
-.\\  {
+.- {
   color: red;
 }
 
 /* class="😀 😓" */
-.\\ .\\  {
+.-.- {
   color: red;
 }
 
 /* class="😀" > class="😓" */
-.\\  .\\  {
+.- .- {
   color: red;
 }
 
 .\\1F600  .a .\\1F600 {
   color: red;
 }
 
-.\\  .\\  .\\  {
+.- .- .- {
   color: red;
 }
 
-div:not(.\\ ) {
+div:not(.-) {
   color: red;
 }
 
-.\\  .b {
+.- .b {
   color: red;
 }
 
-.b .\\  {
+.b .- {
   color: red;
 }
 
-.\\1F613  .\\  {
+.\\1F613  .- {
   color: red;
 }
 
-.\\1F613 .\\  {
+.\\1F613 .- {
   color: red;
 }
 
-.\\  > .\\  > .\\  {
+.- > .- > .- {
   color: red;
 }
 ",
```

Comment:
Let's add a test case
```css
a-b { }
a\C20b { } /* i.e. with spaces */
```

The main problem we need to generate different CSS selectors for such case.

Even more - I think we should replace each control character on own unique symbol, we can generate it from code the character, so even special characters will always generate and unique selector. What do you think?

Reply:
> That could work as expected, but is that common to have different white spaces in a single project?

Usually no, but I think we can solve it more universal for any exotic cases, usually developers don't use such characters in path and in CSS selectors too :smile: 

I think we can just generate use number of a character... yeah it will create more longer CSS selectors, maybe we can use some compression algorithm here to get less length

Reply:
> usually developers don't use such characters in path and in CSS selectors too

I got this issue because we installed a common package in global mode. What means whenever `css-loader` is installed as global, it will be an issue for macOS at least with `Application Support` dir.

Reply:
@alexander-akait Sorry I was busy these days, please help to review again.

Reply:
Friendly ping @alexander-akait

Reply:
@JounQin Sorry for delay too, will review soon